.github/workflows/angular-builders.yml

```diff
@@ -0,0 +1,46 @@
+name: angular-builders
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: true
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/cache@v1
+        id: node-modules
+        with:
+          path: node_modules
```

Comment:
We should probably use multi-path feature for `master` as described [here](https://github.com/actions/cache/blob/master/examples.md#node---lerna).